docs/application/dotnet/guides/app-management/widget-control.md

```diff
@@ -0,0 +1,149 @@
+
+# Widget Control
+
```

Comment:
Please add this sentence back:
The widget control provides information about installing and running widget applications.

Reply:
Done